colin/utils/cont.py

```diff
@@ -0,0 +1,285 @@
+"""
+This is a temporary module to support unpriv way of interacting with container images.
+
+It will be migrated to conu sooner or later.
+"""
+import json
+import os
+import logging
+import shutil
+import subprocess
+from tempfile import mkdtemp
+
+from colin.core.exceptions import ColinException
+
+
+logger = logging.getLogger(__name__)
+
+
+def extract_file_from_tarball(tarball_path, file_path, wd):
+    """
+    Extract selected file (file_path) from tarball (tarball_path) within a selected
+    directory (wd).
+    """
+    tar_cmd = ["tar", "-xf", tarball_path, file_path]
+    out = subprocess.check_output(tar_cmd, cwd=wd)
```

Comment:
What about to log also error in case `image` is corrupted/damaged?

Reply:
Good point, I replaced all subprocess calls with a single function which logs everything.